samples/basic/minimal/common.conf

```diff
@@ -0,0 +1,35 @@
+# Drivers and peripherals
+CONFIG_I2C=n
+CONFIG_WATCHDOG=n
+CONFIG_GPIO=n
+CONFIG_SPI=n
+CONFIG_SERIAL=n
+CONFIG_FLASH=n
+
+# Power management
+CONFIG_SYS_POWER_MANAGEMENT=n
+
+# Interrupts
+CONFIG_DYNAMIC_INTERRUPTS=n
+CONFIG_IRQ_OFFLOAD=n
+
+# Memory protection
+#CONFIG_ARM_MPU=n
+CONFIG_MEMORY_PROTECTION=n
```

Comment:
No need to set CONFIG_MEMORY_PROTECTION, if ARM_MPU is off (MEMORY_PROTECTION is selected by ARM_MPU)

Reply:
This is common, I don't know how x86 does it

Reply:
This won't help you much. In any case you need to make sure you don't have the MPU switches (ARM_MPU, ARC_MPU and X86_MMU).

Reply:
right, I suspected that, thanks for the info